apps/xlayers/src/app/supported-files.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Object that contains information about file types xlayers is compatible with
+ */
+const compatibleFiles = [
+  {
+    name: 'Sketch App',
+    extension: '.sketch',
+    logo: '/assets/supported/sketch.svg',
+    supportedVersions: [49, 50, 51, 52, 53, 59],
```

Comment:
The `supportedVersions` can be computed from the environment config (see [environment.ts](https://github.com/xlayers/xlayers/blob/main/apps/xlayers/src/environments/environment.ts)).